config.js

```diff
@@ -15,5 +15,6 @@ module.exports = {
     secret: process.env.KEYCLOAK_SECRET,
     profile: process.env.API_URL,
     permissions: process.env.PERMISSIONS_SERVICE
-  }
+  },
+  verboseErrors: process.env.VERBOSE_ERRORS
```

Comment:
I'd make this `process.env.VERBOSE_ERRORS === 'TRUE'` because environment variables are _always_ strings, so otherwise `process.env.VERBOSE_ERRORS = 'FALSE'` will make it true, and that's probably bad.

Reply:
caps?